### DISCUSSION TITLE
Implement way to upload local repo remotely on success

### DISCUSSION THREAD

**Discussion by Scrub000** [garethgeorge/backrest#438] [Q&A] [answered]
Is there a way to say, use rclone to upload the local repository to a remote place such as BackBlaze B2, on a successful repo check operation? (Docker)

**Comment by garethgeorge** [comment]
Hey, yes -- definitely possible, you can use a Command Hook configured on `CONDITION_CHECK_SUCCESS` if you'd like to script this.

**Reply by Scrub000** [reply]
Is it possible to call rclone in the Docker container

**Reply by garethgeorge** [reply, marked as answer]
Yes -- the default docker image bundles a number of dependencies https://github.com/garethgeorge/backrest/blob/main/Dockerfile.alpine#L2

Possibly worth mentioning that my recommendation would be to simply add a second repo and plan for your remote rclone storage (not sure how reliable rsync'ing a restic repo to remote storage will be). Could be worth getting some advice on the restic forums re: what you're trying to accomplish.

**Reply by robflate** [reply]
@Scrub000 Did you ever find a solution to this?

**Reply by robflate** [reply]
@garethgeorge 

I'm running a plan that uses a local repo. It stops several docker containers using `HOOKS` and then restarts them after the snapshot is finished. I also want to back this up to Google Drive but I don't want to repeat the process of stopping and starting the docker containers, I just want to sync the local repo to Google Drive.

> Possibly worth mentioning that my recommendation would be to simply add a second repo and plan for your remote rclone storage (not sure how reliable rsync'ing a restic repo to remote storage will be)

Can you elaborate on what you mean here and how it would apply to my situation. Thanks.
